umake/frameworks/baseinstaller.py

```diff
@@ -237,34 +246,34 @@ def get_progress(self, progress_download, progress_requirement):
             self.last_progress_requirement = progress_requirement
 
         # we wait to have the file size to start getting progress proportion info
-
-        # try to compute balance requirement
         if self.balance_requirement_download is None:
-            if not self.pkg_to_install:
-                self.balance_requirement_download = 0
-                self.last_progress_requirement = 0
-                if self.last_progress_download is None:
-                    return
-            else:
-                # we only update if we got a progress from both sides
-                if self.last_progress_download is None or self.last_progress_requirement is None:
-                    return
-                else:
-                    # apply a minimum of 15% (no download or small download + install time)
-                    self.balance_requirement_download = max(self.pkg_size_download /
-                                                            (self.pkg_size_download + self.total_download_size),
-                                                            0.15)
-
+            if self._compute_balanced_requirement() is None:
+                return
         if not self.pbar.finished:  # drawing is delayed, so ensure we are not done first
             progress = self._calculate_progress()
             self.pbar.update(progress)
 
+    def _compute_balanced_requirement(self):
+        if not self.pkg_to_install:
+            self.balance_requirement_download = 0
+            self.last_progress_requirement = 0
+            if self.last_progress_download is None:
+                return
+        else:
+            # we only update if we got a progress from both sides
+            if self.last_progress_download is None or self.last_progress_requirement is None:
+                return
+            # apply a minimum of 15% (no download or small download + install time)
+            self.balance_requirement_download = max(self.pkg_size_download /
+                                                    (self.pkg_size_download +
+                                                     self.total_download_size),
+                                                    0.15)
+
     def _calculate_progress(self):
         progress = self.balance_requirement_download * self.last_progress_requirement +\
             (1 - self.balance_requirement_download) * self.last_progress_download
         normalized_progress = max(0, progress)
-        normalized_progress = min(normalized_progress, 100)
-        return normalized_progress
+        return min(normalized_progress, 100)
```

Comment:
There was no need to reassign it again. Just a itsy-bitsy (personal) code style fix.